{
	"about": "https://jvndb.jvn.jp/ja/contents/2018/JVNDB-2018-013405.html",
	"title": "Pylon における整数オーバーフローの脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2018/JVNDB-2018-013405.html",
	"description": "Pylon (PYLNT) には、整数オーバーフローの脆弱性が存在します。\r\n\r\n本脆弱性は、CVE-2018-11812 と関連する脆弱性です。",
	"identifier": "JVNDB-2018-013405",
	"references": [
		{
			"text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-18667",
			"source": "CVE",
			"id": "CVE-2018-18667"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2018-18667",
			"source": "NVD",
			"id": "CVE-2018-18667"
		},
		{
			"text": "https://cwe.mitre.org/data/definitions/190.html",
			"id": "CWE-190",
			"title": "整数オーバーフローまたはラップアラウンド(CWE-190)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:pylon-network:pylontoken",
			"version": "2.2",
			"vendor": "Pylon Network",
			"product": "PylonToken"
		}
	],
	"cvss": [
		{
			"version": "2.0",
			"score": "5.0",
			"type": "Base",
			"severity": "Medium",
			"vector": "AV:N/AC:L/Au:N/C:N/I:P/A:N"
		},
		{
			"version": "3.0",
			"score": "7.5",
			"type": "Base",
			"severity": "High",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N"
		}
	],
	"date": "2019-02-20T16:06+09:00",
	"issued": "2019-02-20T16:06+09:00",
	"modified": "2019-02-20T16:06+09:00"
}
